test/api_integration/apis/saved_objects/bulk_create.js

```diff
@@ -72,6 +72,9 @@ export default function({ getService }) {
                   attributes: {
                     title: 'A great new dashboard',
                   },
+                  migrationVersion: {
+                    dashboard: resp.body.saved_objects[1].migrationVersion.dashboard,
```

Comment:
This value will change anytime a new dashboard migration is added. This seems like the best workaround when using `kbn-expect` to prevent tests from failing anytime a new dashboard migration is added.

We should probably "modernize" these integration tests, but it felt like it's not enough of a priority to attempt this right now.

Reply:
Yea, having jest toolbox for FTR tests would we really great. `kbn-expect` feels very old when compared to jest.